molecule/test-helpers/verify_podman.yaml

```diff
@@ -0,0 +1,35 @@
+
+- name: verify podman containers
+  block:
+    - name: Check if podman container exists {{ item }}
+      become: true
+      ansible.builtin.command: podman ps -a --filter name={{item}} --format \{\{.Names\}\}
+      register: containers
+    - name: Assert podman container exists {{ item }}
+      ansible.builtin.assert:
+        that:
+          - containers.rc == 0
+          - containers.stdout_lines | length == 1
+          - item in containers.stdout_lines | first
+        fail_msg: "Podman container {{ item }} does not exist"
+    - name: Check if podman container is running {{ item }}
+      become: true
+      ansible.builtin.command: podman ps --filter name={{item}} --format \{\{.Status\}\}
+      register: running_containers
+    - name: Assert podman container is running {{ item }}
+      ansible.builtin.assert:
+        that:
+          - running_containers.rc == 0
+          - running_containers.stdout_lines | length == 1
+          - "'Up' in running_containers.stdout_lines | first"
+        fail_msg: "Podman container {{ item }} is not running"
+    - name: Check the container stdout log exists {{ item }}
+      become: true
+      ansible.builtin.stat:
+        path: "/var/log/containers/stdouts/{{ item }}.log"
+      register: log_exists
+    - name: Assert the container stdout log exists {{ item }}
+      ansible.builtin.assert:
+        that:
+          - log_exists.stat.exists
+        fail_msg: "Container stdout log for {{ item }} does not exist"
```

Comment:
What's the iterator here? Using `{{ item }}` but no `loop:`?

Reply:
you cannot use loop with block, this is a well know limitation in ansible
so the convention is to put the blocks in separate task files and then invoke that in a loop

see the verify.yml for where this is used